drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorPresenter.java

```diff
@@ -394,6 +394,10 @@ protected void refreshModelContent(SimulationRunResult newData) {
         this.lastRunResult = newData;
     }
 
+    public void sendNotification(String text, NotificationEvent.NotificationType type) {
```

Comment:
Why `public`? Do you want to promote this mechanism where the presenter fires notifications for everyone?

Reply:
@danielezonca Yes, I use it inside `KogitoWrapper` class. I think it's safer that exposing the`eventBus`.

Reply:
Actually I think it is already exposed via `getEventBus()` :)
Btw I agree with you that it is better to hide it. We could probably review `getEventBus()` in a future ticket, it should be nice to remove this getter